public/pages/Emails/components/tables/SESSendersTable.tsx

```diff
@@ -104,6 +105,9 @@ export class SESSendersTable extends Component<
     if (!_.isEqual(prevQuery, currQuery)) {
       await this.refresh();
     }
+    if (isDataSourceChanged(this.props, prevProps)) {
```

Comment:
Nitpick:
This check could be worked into line check on line 125 by changing the check to
```
if (!_.isEqual(prevQuery, currQuery) || isDataSourceChanged(this.props, prevProps)) { ...